browser.js

```diff
@@ -0,0 +1,54 @@
+import updater from './updater'
+import epic from './epic'
+import { addLocaleData } from 'react-intl'
+import en from 'react-intl/locale-data/en'
+import routes from './routes'
+import style from './style'
+import Layout from './app/containers/Layout'
+
+const createBrowserEntry = require('dogstack/browser')
+const authentication = require('dogstack-agents/client')
+const Config = require('dogstack/config')
+const config = Config()()
+window.config = config
+
+const store = {
+  updater,
+  epic
+}
+
+const client = {
+  services: [
+    authentication
+  ],
+  apiUrl: config.api.url
+}
+
+// root
+const appNode = document.createElement('div')
```

Comment:
could this go in the `index.html`? i think i was the one who did this, no idea why, probably to move quickly and break things.